src/libraries/Native/Unix/System.Native/pal_autoreleasepool.m

```diff
@@ -42,3 +42,9 @@ void SystemNative_DrainAutoreleasePool(void* pool)
 {
     [((NSAutoreleasePool*)pool) drain];
 }
+
+void SystemNative_AllocateThreadlocalAutoreleasePool(void)
```

Comment:
This does the same thing as `SystemNative_CreateAutoreleasePool`. Why do we need two different methods?

Reply:
No real reason. This was created when I thought I needed to do some TLS related operations. I will remove this one.